data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/consumer/impl/OrderedConsumerVerticle.java

```diff
@@ -122,7 +122,7 @@ void recordsHandler(KafkaConsumerRecords<String, CloudEvent> records) {
 
   void enqueueRecord(TopicPartition topicPartition, KafkaConsumerRecord<String, CloudEvent> record) {
     this.recordDispatcherExecutors.computeIfAbsent(topicPartition, (tp) -> new OrderedAsyncExecutor())
-      .offer(() -> this.recordDispatcher.apply(record).onComplete(v -> this.pendingRecords--));
+      .offer(() -> this.recordDispatcher.dispatch(record).onComplete(v -> this.pendingRecords--));
```

Comment:
I actually do like the more specific term `dispatch`, over the generic `Function.apply`